packages/s/skopeo/abi_used_symbols

```diff
@@ -132,31 +132,8 @@ libc.so.6:unlink
 libc.so.6:unsetenv
 libc.so.6:unshare
 libc.so.6:utimes
-libc.so.6:vasprintf
 libc.so.6:vfprintf
 libc.so.6:write
-libdevmapper.so.1.02:dm_cookie_supported
-libdevmapper.so.1.02:dm_get_library_version
-libdevmapper.so.1.02:dm_get_next_target
-libdevmapper.so.1.02:dm_log_with_errno_init
-libdevmapper.so.1.02:dm_set_dev_dir
-libdevmapper.so.1.02:dm_task_add_target
-libdevmapper.so.1.02:dm_task_create
-libdevmapper.so.1.02:dm_task_deferred_remove
-libdevmapper.so.1.02:dm_task_destroy
-libdevmapper.so.1.02:dm_task_get_deps
-libdevmapper.so.1.02:dm_task_get_driver_version
-libdevmapper.so.1.02:dm_task_get_info
-libdevmapper.so.1.02:dm_task_run
-libdevmapper.so.1.02:dm_task_set_add_node
-libdevmapper.so.1.02:dm_task_set_cookie
-libdevmapper.so.1.02:dm_task_set_message
-libdevmapper.so.1.02:dm_task_set_name
-libdevmapper.so.1.02:dm_task_set_ro
-libdevmapper.so.1.02:dm_task_set_sector
-libdevmapper.so.1.02:dm_udev_get_sync_support
-libdevmapper.so.1.02:dm_udev_set_sync_support
-libdevmapper.so.1.02:dm_udev_wait
```

Comment:
This chunk being removed means either a) pkgconfig(devmapper) is probably no longer needed as a build dep, or b) the build dep is not being used when it should. Can you poke around to see what the situation is?

Reply:
Looks like it was [removed](https://github.com/containers/skopeo/pull/2283)

Reply:
Perfect, thanks for digging that up.